CMakeLists.txt

```diff
@@ -169,6 +157,15 @@ message(STATUS "GEOS: Require C++${CMAKE_CXX_STANDARD}")
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+#-----------------------------------------------------------------------------
+# Targets variable
+#-----------------------------------------------------------------------------
+set(_targets "")
+
+macro(GEOS_ADD_TARGET _name)
```

Comment:
Given there's only two items in this list, I find the indirection actually more harmful than helpful... when looking at the set_target_properties call I have to go back and figure out "now what targets are being set, exactly?".

Reply:
The reason I change it to `set_target_properties` is that it won't pollute other projects' setting of output directories, including library, runtime, archive.
And yes it's not clearer than before, but I think the reason above is more important.
About the question `now what targets are being set, exactly?`, I think it may be better to include all targets?